### ISSUE TITLE
[How to] fine-tune MAGNeT with custom data?

### ISSUE THREAD

**Issue by mahimairaja** [facebookresearch/audiocraft#415]
Hi all

Please answer if you can help

- Can I finetune the MAGNeT Music Generation Model
- If so, What are the things I need like - data format, compute, etc.
- What else to consider?

**Comment by galfaroth** [comment]
I'm also highly interested in these questions. Preferably some google colab that shows the process.

**Comment by Frei-2** [comment]
@mahimairaja Hello, can I ask if you start or finish finetuning MAGNeT? I encounter a lot of issues when executing the musicgen finetuning command. The docs is really hard to follow. I notice that the finetuning commands of MAGNeT and musicgen are quite similar so seek help here.
